### ISSUE TITLE
:set background=dark is unusable.

### ISSUE THREAD

**Issue by linsomniac** [neovim/neovim#3600]
I have white text on black background, and if I run neovim and do ":set background=dark" and then ":help background", some of the text is shown as black.  For example, the first sentence of the results of that start with: When set to "    ", 

If I run the same thing on a vim setup with no .vimrc, I get syntax highlighting off, but at least I can read that it says "When set to "dark",".

**Comment by Grimy** [comment]
This sounds like a problem with your colrscheme. Can you reproduce this with `nvim -u NONE` ?

**Comment by linsomniac** [comment]
I'm having problems reproducing this now.  Before, I didn't have an user rc files, but perhaps I have run neovim in the past and the only file I saw it reading that wasn't from the neovim debian package was:

/home/sean/.local/share/nvim/shada/main.shada

I removed this, re-installed neovim, and did, I believe, everything I did before that produced the missing text, but now it is not.  Maybe I had run a older version of neovim, it produced that main.shada file, which confused neovim or something?  Not sure, but I can't reproduce it, so closing.
